testing/tests/DevExpress.ui.widgets.editors/datebox.mask.tests.js

```diff
@@ -354,6 +354,28 @@ if(devices.real().deviceType === "desktop") {
             assert.notOk($contouredDate.is($selectedDate), "Contoured date isn't a selected");
         });
 
+        test("keyboard mask handlers should work for delete (T832885)", (assert) => {
```

Comment:
'should work' looks a quite general to mee. Can we rename to something more specific?

Reply:
Thanks, it's specified now:  "mask handler should be used instead of the default for delete key when widget is opened"